packages/react/spec/useActionFormNested.spec.tsx

```diff
@@ -2177,6 +2177,62 @@ describe("useActionFormNested", () => {
                       },
                     },
                   },
+                  {
+                    node: {
+                      id: "11",
```

Comment:
This updated test fails when I bring back the old `substring` approach because I introduce edge items with more than one digit.